docs/getting-started/mia-platform-overview.md

```diff
@@ -12,7 +12,7 @@ With Mia-Platform, you can **standardize and reuse your code**, enabling you to
 
 To further foster composability, Mia-Platform features a **software catalog** full of ready-to-use components that you can plug into your software. Among these components, one of the most important is Mia-Platform Fast Data, a data management layer that can be used to build a **Digital Integration Hub**. Thanks to this solution, you can connect your cloud-native platform with existing systems, decouple and offload legacy systems, and serve real-time data 24/7. Thus, you can fully benefit from the [true power of your data](#unleash-the-true-power-of-your-data).
 
-Mia-Platform products are built **by developers for developers**, and you can [actively contribute](#more-resources). Our main purpose is to streamline the software development lifecycle, and we do so by collecting all the tools you need in a single place.
+Mia-Platform products are built **by developers for developers**, and you can [actively contribute](https://docs.mia-platform.eu/#:~:text=Create%20Public%20Variables-,Useful%20resources,-Here%20you%20can). Our main purpose is to streamline the software development lifecycle, and we do so by collecting all the tools you need in a single place.
```

Comment:
This link does not comply with other internal documentation links, which should not contain the domain.

Also it used to link to a specific anchor while now it may highlight text on certain browser, why this change?